### PR TITLE
#39 Inspect a DICOM directory and create appropriate model instances

### PR DESCRIPTION
**Pull request by WGierke**
Get or create an `ImageSeries` instance given the path to a DICOM directory.

## Description
I added the class function `get_or_create` to the `ImageSeries` class.

## Reference to official issue
This should solve #39 

## Motivation and Context
The `ImageSeries` model needs to know the data it's holding.

## How Has This Been Tested?
I wrote a test for it.

## CLA
- [X] I have signed the CLA; if other committers are in the commit history, they have signed the CLA as well

### CODE REVIEW

File: interface/backend/images/models.py [hunk 1]
```diff
@@ -11,6 +13,22 @@ class ImageSeries(models.Model):
...
+    def get_or_create(uri):
+        """
+        Given the absolute uri to a directory with DICOM images of a patient,
+        look up the ImageSeries with the same PatientID and SeriesInstanceUID.
+        If none exists so far, create one.
+        Return a tuple of (ImageSeries, created), where created is a boolean specifying whether the object was created.
+        :return: (ImageSeries, bool)
+        """
+        file_ = glob.glob1(uri, '*.dcm')[0]
+        plan = dicom.read_file(uri + "/" + file_)
```
Comment (lamby) [inline interface/backend/images/models.py]: ``os.path.join``? Or even Django's ``safe_join`` rather..  Oherwise we could have some fun with ``../../../../lol`` I fear

Comment (WGierke) [reply interface/backend/images/models.py]: Very good point, thank you @lamby ! I updated it with `safe_join`.

File: interface/backend/images/models.py [hunk 2]
```diff
@@ -23,3 +47,4 @@ class ImageLocation(models.Model):
     y = models.PositiveSmallIntegerField(help_text='Voxel index for Y axis, zero-index, from top left')
 
     z = models.PositiveSmallIntegerField(help_text='Slice index for Z axis, zero-index')
+
```
Comment (lamby) [inline interface/backend/images/models.py]: Please be careful of checking your commits before pushing... You've read about Van Halen's brown M&M's ider, right? :)

**Comment by lamby** [comment]
thanks @WGierke !

**Pull request merged**